node_modules/gh-orgunit/lib/api.js

```diff
@@ -577,6 +580,145 @@ var deleteOrgUnitSeries = module.exports.deleteOrgUnitSeries = function(ctx, id,
 };
 
 
+//////////////
+// Calendar //
+//////////////
+
+/**
+ * Get the calendar for an organisational unit
+ *
+ * @param  {Context}        ctx                 Standard context containing the current user and the current app
+ * @param  {Number}         id                  The id of the organisational unit to get the calendar for
+ * @param  {String}         start               The timestamp (ISO 8601) from which to get the calendar for the organisation unit
```

Comment:
Actually, that makes sense. However, we do seem to have quite a few places where the ISO 8601 timestamp has been documented as `number`. Maybe we can fix those up?